doc/modules/changes/20200214_clevenger

```diff
@@ -0,0 +1,3 @@
+New: Geometric Multigrid now supports computations with Picard iteration.
```

Comment:
```suggestion
New: The geometric multigrid preconditioner for the Stokes system now supports Picard iterations for the defect correction form of the Stokes equations.
```

The original message was somewhat misleading, because as far as I can tell Picard iterations for the original form of the Stokes equations were already supported beforehand.

Reply:
Yes, this is correct!